napari/layers/_labels_layer/model.py

```diff
@@ -251,7 +251,7 @@ def mode(self, mode):
             self.help = ('hold <space> to pan/zoom, '
                          'click to pick a label')
         elif mode == Mode.PAINT:
-            self.cursor_size = self.brush_size/self._get_rescale()
+            self.cursor_size = self.brush_size/self.scale_factor
```

Comment:
```suggestion
            self.cursor_size = self.brush_size / self.scale_factor
```
also may want to abstract this at some point

Reply:
fixed